english/matrix/matrix_test/matrix_test.md

```diff
@@ -0,0 +1,7 @@
+As you rise out of the marshmallows carcass,
+you are met by a bald black man,
+in his hands he has two pills.
+Which will you choose?
+
+[Choose the blue pill](blue_pill/blue_pill.md)
+[Choose the red pill](red_pill/red_pill.md)  
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.